composer.json

```diff
@@ -17,10 +17,11 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "symfony/config": "^3.4|^4.0|^5.0",
-        "symfony/dependency-injection": "^3.4|^4.0|^5.0",
-        "symfony/http-kernel": "^3.4|^4.0|^5.0",
-        "symfony/mercure": "^0.2"
+        "symfony/config": "^3.4|^4.3|^5.0",
```

Comment:
```suggestion
        "symfony/config": "^3.4|^4.0|^5.0",
```

Reply:
4.x releases prior to 4.3 are not maintained anymore, not sure we should care